src/h_vialib/secure/nonce.py

```diff
@@ -0,0 +1,29 @@
+"""Create values we can check we created."""
+
+from uuid import uuid4
+
+from h_vialib.secure.token import SecureToken
+
+
+class RandomSecureNonce(SecureToken):
+    """A random token we can verify we created with an expiry time.
+
+    This token should be different every time you create it.
+    """
+
+    def create(self, expires=None, max_age=None):  # pylint: disable=arguments-differ
+        """Create a random nonce which changes every time."""
+        return super().create(
+            payload={"salt": uuid4().hex}, expires=expires, max_age=max_age
+        )
+
+
+class RepeatableSecureNonce(SecureToken):
+    """A token which we can verify we created with an expiry time.
+
+    This token should be the same for the same expiry time.
+    """
+
+    def create(self, expires=None, max_age=None):  # pylint: disable=arguments-differ
+        """Create a random nonce which is the same for the same expiry."""
```

Comment:
```suggestion
        """Return a random nonce. Each call with the same args returns the same nonce."""
```

Reply:
This isn't quite true. If you set the same `max_age`, it's not going to be the same. This makes me think that maybe this shouldn't support `max_age`.